server/test/testDBTablesUser.cc

```diff
@@ -498,6 +498,19 @@ void test_deleteUser(void)
 	assertUsersInDB(userIdSet);
 }
 
+void test_deleteMyself(void)
+{
+	DECLARE_DBTABLES_USER(dbUser);
+	OperationPrivilege privilege(ALL_PRIVILEGES);
+	const UserIdType targetId = privilege.getUserId();
+	HatoholError err = dbUser.deleteUserInfo(targetId, privilege);
+	assertHatoholError(HTERR_DELETE_MYSELF, err);
+
+	UserIdSet userIdSet;
```

Comment:
These these lines are unnecessary, aren't they ?
The first half of the test seems not to delete a user at all.